### DISCUSSION TITLE
This weekend I will Release Multiple Picker version 2.0 🚀

### DISCUSSION THREAD

**Discussion by baronha** [NitrogenZLab/react-native-multiple-image-picker#182] [Announcements]
# Version 2.0 🚀

As mentioned before. 
React Native Multiple Image Picker will be released with many new improvements:
- Use [Nitro Module](https://nitro.margelo.com/docs/nitro-modules) instead of Turbo Module.
- Use [HXPhotoPicker](https://github.com/SilenceLove/HXPhotoPicker) instead of TLPhotoPicker like the old version.
- New Arch support due to Nitro Modules support
- Multiple Cropping
- Dynamic Themes.
- Support Language:
  - `system`: 🌐 System default
  - `zh-Hans`: 🇨🇳 Simplified Chinese
  - `zh-Hant`: 🇹🇼 Traditional Chinese
  - `ja`: 🇯🇵 Japanese
  - `ko`: 🇰🇷 Korean
  - `en`: 🇬🇧 English
  - `vi`: 🇻🇳 Vietnamese
  - `ru`: 🇷🇺 Russian
  - `de`: 🇩🇪 German
  - `fr`: 🇫🇷 French
  - `ar`: 🇸🇦 Arabic

  **iOS Only**:

  - `id`: 🇮🇩 Indonesian
  - `th`: 🇹🇭 Thai

And many other improvements. 
PR: #180 

## Video Demo
https://github.com/user-attachments/assets/79580bc7-237c-46b7-b92e-1479cc6d9079

**Comment by amosaxe** [comment]
finally, awesome work, thanks for the hard work.

**Reply by baronha** [reply]
What I should do to help the community become stronger 💪

**Comment by baronha** [comment]
Actively opening cropped images will also be available in an upcoming version.
https://github.com/baronha/react-native-multiple-image-picker/pull/184

**Comment by baronha** [comment]
Now you can completely see Image/Video called from javascript
https://github.com/baronha/react-native-multiple-image-picker/pull/186

https://github.com/user-attachments/assets/92a1c319-ad99-4d24-892c-7cd878787acd
